common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -30,22 +30,46 @@ object ImageMetadataConverter {
       dateTaken           = (fileMetadata.exifSub.get("Date/Time Original Composite") flatMap parseRandomDate) orElse
                             (fileMetadata.iptc.get("Date Time Created Composite") flatMap parseRandomDate) orElse
                             (fileMetadata.xmp.get("photoshop:DateCreated") flatMap parseRandomDate),
-      description         = fileMetadata.iptc.get("Caption/Abstract"),
-      credit              = fileMetadata.iptc.get("Credit"),
-      byline              = fileMetadata.iptc.get("By-line"),
-      bylineTitle         = fileMetadata.iptc.get("By-line Title"),
-      title               = fileMetadata.iptc.get("Headline"),
-      copyrightNotice     = fileMetadata.iptc.get("Copyright Notice"),
+      description         = fileMetadata.xmp.get("dc:description[1]") orElse
+                            fileMetadata.iptc.get("Caption/Abstract") orElse
+                            fileMetadata.exif.get("Image Description"),
+      credit              = fileMetadata.xmp.get("photoshop:Credit") orElse
+                            fileMetadata.iptc.get("Credit"),
+      // FIXME: Have a way of dealing with arrays, like [1] here. Not much of a practical problem in our corpus, but still..
```

Comment:
If we can find some examples of `xmp` arrays then we could come up with some logic around which element of the array we should pick. I suspect our problem will be if there are multiple people in the byline they might appear in different array elements and as such we'll just drop everyone except the first with this code.

Reply:
There are some arrayed fields in XMP (and structured ones, too). I have checked and taking just the first here is quite safe given the quality of data. But it is not correct and may bite us one day. Writing it properly (incl. deduping of repeats) is beyond my skillz. Let’s hope we do it one day.